src/commands/config/list.js

```diff
@@ -1,19 +1,27 @@
 const { BaseCommand } = require('@twilio/cli-core').baseCommands;
 
-const { availableConfigs, getFromEnvironment } = require('../../services/config-utility');
+const { getFromEnvironment } = require('../../services/config-utility');
 
 class ConfigList extends BaseCommand {
   async run() {
     await super.run();
     const configData = [];
-    availableConfigs.forEach((config) => {
+    const parseUserConfigValue = (config) => {
+      // only parse to JSON string in case of Objects/ Arrays
+      if (this.userConfig[config] instanceof Array || this.userConfig[config] instanceof Object) {
+        return JSON.stringify(this.userConfig[config]);
+      }
+      return this.userConfig[config];
+    };
```

Comment:
[Q] Does it have to be this verbose? From what I understand, `JSON.stringify()` handles Strings, undefined, null vars gracefully. I might be missing something here, just checking in. Could be simplified to : 

```
const parseUserConfigValue = (config) => JSON.stringify(this.userConfig[config]);
```